.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```

Comment:
It might be easier for the users to paste the tool's output, so we can ask exactly that, instead of asking for a screenshot which might be harder to make.

Reply:
We can also remind the user that one can write monospaced font by enclosing the text in triple ticks `.

Reply:
91b91b71e0935ecdc5005c4815e11b9a14a734be